.github/workflows/release.yml

```diff
@@ -34,11 +34,11 @@ jobs:
           # (optional) On which platform to distribute the `.tar.gz` file.
           # [default value: unix]
           # [possible values: all, unix, windows, none]
-          tar: all
+          tar: unix
           # (optional) On which platform to distribute the `.zip` file.
           # [default value: windows]
           # [possible values: all, unix, windows, none]
```

Comment:
I think that this comment is now outdated and should be updated.

Reply:
May I also ask why you removed the `zip` option? Why not distribute both the .tar.gz and the .zip archives for all platforms?

Reply:
Thanks for catching that!

I added back the zip option. And also added a comment for the archive option.